src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -12592,5 +12592,1268 @@ public void ScopedReserved_Alias_Escaped()
+            // https://github.com/dotnet/roslyn/issues/62691: Expected errors should be reported when 'ref R<T>' is considered 'scoped ref'.
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void UnscopedRefAttribute_Implementations_01()
+        {
+            var source =
+@"using System.Diagnostics.CodeAnalysis;
+interface I<T>
+{
+    void F1(out T t);
+    void F2([UnscopedRef] out T t);
+}
+class C1 : I<int>
+{
+    public void F1(out int i) { i = 0; }
+    public void F2([UnscopedRef] out int i) { i = 0; }
+}
+class C2 : I<int>
+{
+    public void F1([UnscopedRef] out int i) { i = 0; } // 1
+    public void F2(out int i) { i = 0; } // 2
+}
+class C3 : I<object>
+{
+    void I<object>.F1(out object o) { o = null; }
+    void I<object>.F2([UnscopedRef] out object o) { o = null; }
+}
+class C4 : I<object>
+{
+    void I<object>.F1([UnscopedRef] out object o) { o = null; } // 3
+    void I<object>.F2(out object o) { o = null; } // 4
+}
+";
+            var comp = CreateCompilation(new[] { source, UnscopedRefAttributeDefinition });
+            comp.VerifyDiagnostics(
+                // (14,17): error CS8987: The 'scoped' modifier of parameter 'i' doesn't match overridden or implemented member.
+                //     public void F1([UnscopedRef] out int i) { i = 0; } // 1
+                Diagnostic(ErrorCode.ERR_ScopedMismatchInParameterOfOverrideOrImplementation, "F1").WithArguments("i").WithLocation(14, 17),
+                // (15,17): error CS8987: The 'scoped' modifier of parameter 'i' doesn't match overridden or implemented member.
+                //     public void F2(out int i) { i = 0; } // 2
+                Diagnostic(ErrorCode.ERR_ScopedMismatchInParameterOfOverrideOrImplementation, "F2").WithArguments("i").WithLocation(15, 17),
+                // (24,20): error CS8987: The 'scoped' modifier of parameter 'o' doesn't match overridden or implemented member.
+                //     void I<object>.F1([UnscopedRef] out object o) { o = null; } // 3
+                Diagnostic(ErrorCode.ERR_ScopedMismatchInParameterOfOverrideOrImplementation, "F1").WithArguments("o").WithLocation(24, 20),
+                // (25,20): error CS8987: The 'scoped' modifier of parameter 'o' doesn't match overridden or implemented member.
+                //     void I<object>.F2(out object o) { o = null; } // 4
+                Diagnostic(ErrorCode.ERR_ScopedMismatchInParameterOfOverrideOrImplementation, "F2").WithArguments("o").WithLocation(25, 20));
+        }
+
+        [Fact]
+        public void UnscopedRefAttribute_Implementations_02()
+        {
+            var source =
+@"using System.Diagnostics.CodeAnalysis;
+interface I<T>
+{
+    void F1(ref T t);
+    void F2([UnscopedRef] ref T t);
+}
+class C1 : I<int>
+{
+    public void F1(ref int i) { }
+    public void F2([UnscopedRef] ref int i) { }
+}
+class C2 : I<int>
+{
+    public void F1([UnscopedRef] ref int i) { } // 1
```

Comment:
Let's open follow-up issue for variance in OHI

Reply:
Can be bundled with misapplied [ScopedRef] follow-up (bullet 3 in list of test cases)

Reply:
> Let's open follow-up issue for variance in OHI

Added links to the existing variance issue.